scripts/preconversion_assessment_script.py

```diff
@@ -22,16 +23,18 @@ def __init__(self, path="", host=""):
 
 class ProcessError(Exception):
     """Custom exception to report errors during setup and run of conver2rhel"""
+
     def __init__(self, message):
         super(ProcessError, self).__init__(message)
         self.message = message
 
 
 class OutputCollector(object):
     """Wrapper class for script expected stdout"""
+
     def __init__(self, status="", message="", report="", report_json=None):
         if report_json is None:
-            report_json = {}
+            report_json = {"entries": []}
```

Comment:
I think we can just initialize this as being an empty dict, as you already covered how that return value should appear in the report_json. 

```suggestion
            report_json = {}
```

Specially because if that report_json changes in the future, we will need to update it here too. It is just initializing to a default value, so it will not become None anyways.

Reply:
ok, makes sense